introduction.md

```diff
@@ -1,2 +1,4 @@
 Hello fellow dojo people, my name is Pei Pengjun, you call call me PJ, very happy to be here. 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?